guides/doc-Installing_Proxy_on_Debian/master.adoc

```diff
@@ -3,12 +3,13 @@
 :numbered:
 :toc:
 :toc-placement: left
-
 include::common/attributes.adoc[]
-
 :context: {smart-proxy-context}
 :ProductName: {SmartProxyServer}
```

Comment:
In asciidoc header must not contain empty lines. I fixed this here, althought, I haven't seen any differences in the output.